### ISSUE TITLE
Add pages on best git workflows and practices?

### ISSUE THREAD

**Issue by signekb** [seedcase-project/team#34]
Maybe we could benefit from having explicit best-practice guidelines for e.g., good commit messages, issues, prs? 

If you agree, I can spend some time writing drafts for these pages.

**Comment by lwjohnst86** [comment]
Yes! Completely agree, it's been on my to-do list. If you want to start something, that would be fabulous!

**Comment by signekb** [comment]
Great! Will look at it after my vacation 👍

**Comment by lwjohnst86** [comment]
Just linking this reference here on using the slash within git branches to separate things: https://code.erpenbeck.io/git/2021/03/01/git-naming-conventions/

**Comment by lwjohnst86** [comment]
related to commit messages: https://www.conventionalcommits.org/en/v1.0.0-beta.2/

**Comment by signekb** [comment]
Yep, have looked at conventional commits. Should I write that up as the recommended convention for commit messages?
